packages/web-client/pages/_app.tsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
```

Comment:
Why is this disabled?

Reply:
throws an error: 'anything' is missing in props validationeslint(react/prop-types)

Reply:
Choose between the interface solution, or disabling the rule, you are doing both right now.